rpm/kubelet.spec

```diff
@@ -21,14 +21,34 @@ Requires: ethtool
 %description
 The node agent of Kubernetes, the container cluster manager.
 
-%package plugin-cni
+%package -n kubernetes-cni
 
+Version: 0.3.0.1
+Release: %{RPM_RELEASE}.07a8a2
 Summary: Binaries required to provision kubernetes container networking
 Requires: kubelet
 
-%description plugin-cni
+%description -n kubernetes-cni
 Binaries required to provision container networking.
 
+%package -n kubectl
+
+Summary: Command-line utility for interacting with a Kubernetes cluster.
+
+%description -n kubectl
+Command-line utility for interacting with a Kubernetes cluster.
+
+%package -n kubeadm
+
+Version: 1.5.0
+Release: %{RPM_RELEASE}.alpha.gc19e08e
```

Comment:
should be `%{RPM_RELEASE}.alpha.0-1403-gc19e08e`

If `-` are unsupported, then `.alpha.0.1403.gc19e08e`